usvm-ts/src/main/kotlin/org/usvm/machine/TsGraph.kt

```diff
@@ -7,29 +7,36 @@ import org.usvm.dataflow.ts.graph.EtsApplicationGraph
 import org.usvm.dataflow.ts.graph.EtsApplicationGraphImpl
 import org.usvm.statistics.ApplicationGraph
 
-class TsApplicationGraph(scene: EtsScene) : ApplicationGraph<EtsMethod, EtsStmt> {
-    private val applicationGraph: EtsApplicationGraph = EtsApplicationGraphImpl(scene)
+class TsGraph(scene: EtsScene) : ApplicationGraph<EtsMethod, EtsStmt> {
+    private val etsGraph: EtsApplicationGraph = EtsApplicationGraphImpl(scene)
+
+    val cp: EtsScene
+        get() = etsGraph.cp
 
     override fun predecessors(node: EtsStmt): Sequence<EtsStmt> =
-        applicationGraph.predecessors(node)
+        etsGraph.predecessors(node)
 
     override fun successors(node: EtsStmt): Sequence<EtsStmt> =
-        applicationGraph.successors(node)
+        if (node is TsMethodCall) {
+            etsGraph.successors(node.returnSite)
```

Comment:
Why do we need this change? Do we have it in JC version as well?

Reply:
Yes, see `JcApplicationGraph` in usvm-jvm. There, it is implemented via `.originalStmt` extension, which is effectively the same as the code here, where we directly use `if (node is TsMethodCall)`.